apps/condo/domains/acquiring/schema/PaymentStatusChangeWebhookUrl.test.js

```diff
@@ -0,0 +1,342 @@
+/**
```

Comment:
you don't need /** ... */ for tests, any for specs

Reply:
The comment was removed